config/api.yml

```diff
@@ -1848,6 +1850,34 @@
       :post:
       - :name: query
         :identifier: miq_cloud_networks_view
+  :orchestration_templates:
+    :description: Orchestration Template
+    :identifier: orchestration_templates_accord
+    :options:
+    - :collection
+    - :subcollection
```

Comment:
I didn't see the equivalent orchestration_templates being referenced as a subcollection from another collection.